coffeecard/CoffeeCard.Models/DataTransferObjects/v2/Product/AddProductRequest.cs

```diff
@@ -63,6 +63,11 @@ public class AddProductRequest
         [DefaultValue(true)]
         public bool Visible { get; set; } = true;
 
+        /// <summary>
+        /// Gets or sets the user groups that can access the product.
+        /// </summary>
+        /// <value> Product User Groups </value>
+        /// <example> Customer </example>
```

Comment:
This is a list of UserGroups, thus the example should be an array. Please also remember to update the example block above the class declaration